README.helm.md

```diff
@@ -0,0 +1,76 @@
+# Korifi Helm chart
+
+This documents the [Helm](https://helm.sh/) chart for [Korifi](https://github.com/cloudfoundry/korifi).
+
+The chart is a composition of subcharts, one per component, with each individual component configuration nested under a top-level key named after the component itself.
+Values under the top-level `global` key apply to all components.
+Each component can be excluded from the deployment by the setting its `include` value to `false`.
+See [_Customizing the Chart Before Installing_](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing) for details on how to specify values when installing a Helm chart.
+
+Here are all the values that can be set for the chart:
```

Comment:
Other projects seem to be documenting their helm values in tabular form: https://github.com/concourse/concourse-chart. It looks a bit more organized, not sure if we want to do that

Reply:
That looks a bit less readable to me as the table doesn't fit in the page. I'd stick with this format for now and see how it goes.